src/Commands/ShifterToggle.cpp

```diff
@@ -0,0 +1,61 @@
+// RobotBuilder Version: 2.0
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// C++ from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+
+#include "ShifterToggle.h"
+
+// BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+
+ShifterToggle::ShifterToggle(): Command() {
+        // Use requires() here to declare subsystem dependencies
+    // eg. requires(chassis);
+    // BEGIN AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+	Requires(Robot::chassis.get());
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
+}
+
+    // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTRUCTOR
+
+// Called just before this Command runs the first time
+void ShifterToggle::Initialize() {
+
+}
+
+// Called repeatedly when this Command is scheduled to run
+void ShifterToggle::Execute() {
+	//High Speed Transmission
+	if(RobotMap::chassisshiftLeft->Get() == DoubleSolenoid::kForward)
+	{
+		RobotMap::chassisshiftLeft->Set(DoubleSolenoid::kReverse);
```

Comment:
I forgot to mention. We create the actuators using camel case for each subsystems, but if you look at robotmap for some reason it adds the name of the subsystem before causing this interesting issue. The work around is when we name actuators(not just motors but also solenoids) we start with a capital and use camel case.